### PR TITLE
Add rollbar

### PR DESCRIPTION
**Pull request by PirosB3**
Adds rollbar support

### CODE REVIEW

File: pybay/settings.py [hunk 1]
```diff
@@ -105,6 +105,7 @@
     "django.contrib.auth.middleware.SessionAuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "rollbar.contrib.django.middleware.RollbarNotifierMiddleware",
```
Comment (epsy) [inline pybay/settings.py]: Should this be last? We might want to put it earlier if we want to be notified of errors in the django middlewares

File: pybay/settings.py [hunk 2]
```diff
@@ -223,3 +224,16 @@
...
+    rollbar_token = f.read().strip()
+
+ROLLBAR = {
+    'access_token': rollbar_token,
+    'environment': 'development' if DEBUG else 'production',
+    'root': BASE_DIR,
+}
+
+# Intentially added below the ROLLBAR const. Please do not move
+import rollbar
```
Comment (epsy) [inline pybay/settings.py]: Q: Does this import have to be at the bottom?

File: pybay/settings.py [hunk 3]
```diff
@@ -223,3 +224,17 @@
...
+        rollbar_token = f.read().strip()
+
+    ROLLBAR = {
+        'access_token': rollbar_token,
+        'environment': 'development' if DEBUG else 'production',
+        'root': BASE_DIR,
+    }
+
+    # Intentially added below the ROLLBAR const. Please do not move
+    import rollbar
```
Comment (epsy) [inline pybay/settings.py:245]: Q: can we not import this on top?

**Review by epsy — CHANGES_REQUESTED** [review]

File: pybay/settings.py (hunk 3 above)
Comment (PirosB3) [reply pybay/settings.py:245]: no because they need to have access to django settings

File: pybay/settings.py [hunk 4]
```diff
@@ -223,3 +224,17 @@
 CRISPY_TEMPLATE_PACK = 'bootstrap3'
 
 SHOW_SPEAKERS_LIST_NAVBAR_LINK = False
+
+if os.environ.get('TRAVIS', False):
```
Comment (epsy) [inline pybay/settings.py]: you missed a ``not``

**Review by epsy — CHANGES_REQUESTED** [review]

File: pybay/settings.py (hunk 1 above)
Comment (PirosB3) [reply pybay/settings.py]: good idea

**Review by epsy — APPROVED** [review]

**Comment by mzdaniel** [comment]
This rollbar thing look pretty. Thank you Daniel!

Couple of comments:
* Please review the dev environment. I am getting: 
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/data/pybay/pybay/settings.py", line 229, in <module>
    with open('/home/pybay/rollbar.txt') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/pybay/rollbar.txt'
for that conditional.
* Then for merging into prod, lets squash all those commits

**Comment by PirosB3** [comment]
@mzdaniel did all changes

**Comment by mzdaniel** [comment]
Ok. Maybe coincidence but looks like 'branch' attribute is needed on ROLLBAR setting. 
Added that, commits squashed and rebased, and deployed in prod.

**Pull request merged**